src/components/fields/select-users.js

```diff
@@ -122,6 +122,14 @@ class SelectUsers extends React.Component {
 		return usernames.map( this.getOptionFromUsername );
 	}
 
+	ucFirst( str ) {
+		if ( str.length > 0 ) {
+			str = str.charAt( 0 ).toUpperCase() + str.substr( 1 );
```

Comment:
In the future, we might want to use [toLocaleUpperCase()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLocaleUpperCase), but it looks like the standard [toUpperCase()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toUpperCase) will work fine in almost every instance.